.github/workflows/ansible-test.yml

```diff
@@ -7,15 +7,15 @@ on:
     # * is a special character in YAML so you have to quote this string
     - cron: '0 8 * * *'
 env:
-  python_version: '3.10'
+  python_version: '3.11'
 jobs:
   build:
     name: Build collection
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ansible: [v2.15.11, v2.16.6, stable-2.16, stable-2.17]
+        ansible: [v2.16.6, stable-2.16, stable-2.17, stable-2.18]
     steps:
       - name: Check out code
         uses: actions/checkout@v4
```

Comment:
should we raise two issue for nd and mso also and make same changes for versions?

Reply:
Already done, see DCNE-249.